sbhs/views.py

```diff
@@ -163,9 +168,10 @@ def user_register(request):
         return render(request,'account/home.html')
     context={}
     if request.method == 'POST':
-        form = UserRegistrationForm(request.POST)
-        if form.is_valid():
-            u_name, pwd, user_email, key= form.save()
+        registration_form = UserRegistrationForm(request.POST)
+        login_form = UserLoginForm()
```

Comment:
Why is login form needed in registration?
And user is logged in as he registers, so login form is not needed in that case.
If any other purpose then let us know.